src/Symfony/Component/Form/Tests/Extension/Core/Type/DateTypeTest.php

```diff
@@ -1156,6 +1156,62 @@ public function testDateTimeImmutableInputTimezoneNotMatchingModelTimezone()
         ]);
     }
 
+    public function testSubmitWithCustomCalendarOption()
+    {
+        $calendar = \IntlCalendar::createInstance();
+        $calendar->setFirstDayOfWeek(\IntlCalendar::DOW_SUNDAY);
+
+        $form = $this->factory->create(static::TESTED_TYPE, options: self::getFormOptionsWithCalendar($calendar));
```

Comment:
even if that's quite some duplication I would inline the options here instead of calling the function to make reading the tests easier (I would be able to see the actual values without having to jump back and forth in the test class)

Reply:
I understand. I inlined the options everywhere